plugins/ipam/static/main.go

```diff
@@ -161,7 +161,7 @@ func LoadIPAMConfig(bytes []byte, envArgs string) (*IPAMConfig, string, error) {
 
 				ip, subnet, err := net.ParseCIDR(ipstr)
 				if err != nil {
-					return nil, "", fmt.Errorf("invalid CIDR %s: %s", ipstr, err)
+					return nil, "", err
```

Comment:
Maybe make some actionable error message such as:

```
fmt.Errorf("expected CIDR notation, got '%s' with error: %s", ipstr, err)
```

Reply:
@dougbtv your proposal would generate the string `"expected CIDR notation, got '10.10.0.1' with error: invalid CIDR address: 10.10.0.1"` which, imo, is also confusing. 

I don't mind using the first part of the string though. I think that as long as we stop wrapping the error we should be good; golang already gives us a pretty decent error msg w/ `"invalid CIDR address: 10.10.0.1"`

Reply:
Yep, agreed. That definitely adds to the "chart junk" factor to have the IP address repeated like that. Probably more descriptive to use the field name (which I didn't look up, sorry), and I can probably make it a bit more actionable, maybe something like:

fmt.Errorf("The $foo field is expected to be in CIDR notation: %s", err)

Reply:
The thing is that the IP can come from a field named 'ip' when passed via env vars, an array of IPs on a field named 'ips' on both cni args, and runtime config, and finally, an array of 'addresses'  via the ipam struct.